core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -273,6 +278,13 @@ protected void startServingWebServer() {
     mWebServer.start();
   }
 
+  /**
+   * Starts jvm monitor process, to monitor jvm.
+   */
+  protected void startJvmMonitorProcess() {
```

Comment:
do we know how much is the performance overhead by introducing the Jvm Monitor Process? thanks

Reply:
@ifcharming We can not know the accurate performance overhead. BUt we can make a conclusion by log level. I think that is enough.

Reply:
SG